src/components/TransactionsTracker/useTransactionsTracker.ts

```diff
@@ -12,6 +13,8 @@ export interface TransactionsTrackerType {
 }
 
 export function useTransactionsTracker(props?: TransactionsTrackerType) {
+  const timeoutRef = useRef<NodeJS.Timeout>();
```

Comment:
I don't think we need a ref for this, you can keep the reference in a local variable inside useEffect